src/main/java/com/server/RequestReader.java

```diff
@@ -3,27 +3,31 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 
+import static java.lang.Integer.parseInt;
+
 public class RequestReader {
 
     private BufferedReader bufferedReader;
+    private int contentLength;
 
     RequestReader(BufferedReader bufferedReader) throws IOException {
         this.bufferedReader = bufferedReader;
     }
 
     public String getHeader() throws IOException {
-        return readHeader();
+        String header = read("\r\n");
+        String finalResponse = "";
```

Comment:
nit: Maybe we can call this string `requestString` or something so as not to confuse it for any of our `response` stuff...? What do you think?